tasks/system_probe.py

```diff
@@ -960,6 +960,22 @@ def setup_runtime_clang(ctx):
         ctx.run(f"{sudo} chmod 0755 /opt/datadog-agent/embedded/bin/llc-bpf")
 
 
+def setup_system_clang(ctx):
+    sudo = "sudo" if not is_root() else ""
+    clang_res = ctx.run(f"clang --version", warn=True)
+    clang_version_str = clang_res.stdout.split("\n")[0].split(" ")[2].strip() if clang_res.ok else ""
+    if clang_version_str == CLANG_VERSION:
+        return
+    
+    ctx.run(f"{sudo} rm -r /opt/clang", warn=True)
```

Comment:
Can we please not do that ? On my devenv clang is installed by my package manager so I have no idea how this would work with `apt upgrade` and related stuff. I think throwing an error here saying "you are using the wrong version go install the correct one" would be simpler and better for diverse environments

Reply:
Moreover what guarantees that clang is installed in `/opt/clang` ?

Reply:
That is a good point. I took this approach because the provisioning scripts installed clang in `/opt/clang`. But, you're right there isn't any guarantee that it will be so on all systems.